### ISSUE TITLE
Datamodel types without scalar fields break prisma client generation

### ISSUE THREAD

**Issue by ziegfried** [prisma/prisma#3452]
**Describe the bug**
When creating a data model type without any scalar fields (ie. only relation fields), code generation breaks.

**To Reproduce**

1. Create a data model type without scalar fields, eg:

```graphql
type Foobar {
  user: User
}
```

2. Run `prisma generate`

```
$ yarn prisma generate
yarn run v1.10.1
$ /Users/sp/dev/temp/prisma-bug/node_modules/.bin/prisma generate
Generating schema 49ms
 ▸    Syntax Error: Expected Name, found }

Get in touch if you need help: https://www.prisma.io/forum/
To get more detailed output, run $ export DEBUG="*"
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected behavior**
No error running the generate command.

**Versions:**
 - `prisma` CLI: `prisma/1.20.1 (darwin-x64) node-v10.10.0`

**Additional context**

Seems like prisma generates an empty input type in this case:

```graphql
input FoobarUpdateManyMutationInput {

}
```

and the graphql parser doesn't like that.

**Comment by qirex** [comment]
Had the same issue with 1.20.1. Rolled back to 1.19.3 because of this. 
Previous Issues:
* #1981  
* #2123
* #3382

**Comment by guatedude2** [comment]
I'm having the same issue rolled back to 1.20.0 and it worked

**Comment by TheYojimbo** [comment]
Can confirm what @guatedude2 said, can't generate types without scalar fields with prisma@1.20.1, works after rolling back to 1.20.0.

**Comment by timsuchanek** [comment]
This is fixed in `1.20.5` please let us know if that is not the case and we'll reopen it!

**Comment by geroj** [comment]
I have same issue with version 1.21.0. I thought that should work after fix in `1.20.5` @timsuchanek 

`yarn prisma generate` with following datamodel works fine:
`type FeatureSet implements Node {
  id: ID! @unique
  key: String!
  featureList: [Feature!]!
} 

type Feature implements Node{
  id: ID! @unique
  temp: String!
  featureSetList: [FeatureSet!]!
} 
`

While one where temp attribute is excluded have the same issue describe above.
`type FeatureSet implements Node {
  id: ID! @unique
  key: String!
  featureList: [Feature!]!
} 

type Feature implements Node{
  id: ID! @unique
  featureSetList: [FeatureSet!]!
} 
`
